### ISSUE TITLE
Have you considered TokenMix in hidden layers?

### ISSUE THREAD

**Issue by cmsflash** [Sense-X/TokenMix#6]
In addition to TokenMix before the first Transformer block, have you considered or tried TokenMix in the middle of the model?

**Comment by jihaonew** [comment]
I did try it. 
If I understand correctly, it is close to [Manifold Mixup](https://arxiv.org/abs/1806.05236). I believe this will be an interesting extension of CutMix/TokenMix (etc) in the feature space. 
Have you tried CutMix in the feature space?

**Comment by cmsflash** [comment]
Yep, I am indeed effectively talking about Manfold Mixup for TokenMix/CutMix. The Manifold Mixup paper is very interesting and thank you so much for bringing it up. I haven't tried the hidden layer version of CutMix/TokenMix. If you have tried hidden-layer TokenMix, could you share the results?

**Comment by jihaonew** [comment]
OK.

**Comment by cmsflash** [comment]
Great! Do you want to share the results on this issue or somewhere else?

**Comment by cmsflash** [comment]
@jihaonew Did you get any results?
